scripts/common.sh

```diff
@@ -29,16 +29,35 @@ prettierUgly() {
     echo "$acc"
 }
 
-tslintUgly() {
+eslintUgly() {
     local acc=""
     local IFS=$'\n'
     local tmpdir
-    tmpdir=$(mktemp -d "tslint.XXXXXXXXX")
+    tmpdir=$(mktemp -d "eslint.XXXXXXXXX")
+
+    # TODO: need to add this directory to a tsconfig.json
```

Comment:
The pre-commit hook is broken currently - it always says changed files are broken.

We need to either add the temporary directory to the/a tsconfig programmatically once it is created or create a different temporary directory in the source tree that can be picked up by tsconfig (e.g. ./.tmp/ with tsconfig including ./.tmp/**ts)